pkg/brctl/util.go

```diff
@@ -91,52 +91,64 @@ func getIndexFromInterfaceName(ifname string) (int, error) {
 	return int(ifrIfindex), nil
 }
 
-// set values for the bridge
-// all values in the sysfs are of type <bytes> + '\n'
-func setBridgeValue(bridge string, name string, value []byte, _ uint64) error {
-	err := os.WriteFile(filepath.Join(BRCTL_SYS_NET, bridge, "bridge", name), append(value, BRCTL_SYS_SUFFIX), 0)
-	if err != nil {
-		return err
+var ErrBridgeNotExist = errors.New("bridge does not exist")
+
+// setBridgeValue sets the value of a bridge setting in the sysfs.
+func setBridgeValue(bridge string, setting string, value []byte, _ uint64) error {
+	bridgePath := filepath.Join(BRCTL_SYS_NET, bridge)
+	settingPath := filepath.Join(bridgePath, "bridge", setting)
```

Comment:
```suggestion
	settingPath := filepath.Join(bridgePath, BRCTL_BRIDGE_PATH, setting)
```
Does it make sense to also move this into `const.go` or is this overkill?

Reply:
Overkill for now, you could generally argue about code organization (util.go isn't a good name anyway etc.)